src/zenml/cli/stack.py

```diff
@@ -681,3 +714,77 @@ def down_stack(force: bool = False) -> None:
             f"Suspending resources for active stack '{stack_.name}'."
         )
         stack_.suspend()
+
+
+@stack.command("export")
+@click.argument("stack_name", type=str, required=True)
+@click.argument("filename", type=str, required=True)
```

Comment:
Could this be an optional requirement, and if you don't pass it in it'll just use either `stackname.yaml` or `profile-stackname.yaml`?